seed_isort_config.py

```diff
@@ -68,9 +69,10 @@ def main(argv=None):
             break
     else:
         print(
-            'Could not find a `known_third_party` setting in any of '
-            '.isort.cfg, setup.cfg, tox.ini.  '
-            'Set up an initial config and run again!',
+            'Could not find an isort section in any of '
```

Comment:
not sure why the verbiage changed here?

I'd write this:

```python
        print(
            'Could not find a `known_third_party` setting in any of {}.  '
            'Set up an initial config and run again!'.format(
                ', '.join(SUPPORTED_CONF_FILES),
            ),
        )
```

(I see you changing my two-spaces to a single space!)

Reply:
I think I mixed up my changes between branches. You're right this can probably stay as-is. I changed the spacing because sentences should be separated by one space!

> The Complete Manual on Typography (2003) states that "The typewriter tradition of separating sentences with two word spaces after a period has no place in typesetting" and the single space is "standard typographic practice".

https://en.wikipedia.org/wiki/Sentence_spacing